### PR TITLE
Issue #547: Adapted CircuitBreakerConfigurationProperties to support …

### PR DESCRIPTION
**Pull request by RobWin**
…the new sliding window types.

### CODE REVIEW

File: resilience4j-spring-boot2/src/test/resources/application.yaml [hunk 1]
```diff
@@ -27,16 +27,17 @@ resilience4j.circuitbreaker:
   circuitBreakerAspectOrder: 400
   configs:
     default:
-      ringBufferSizeInClosedState: 100
-      ringBufferSizeInHalfOpenState: 10
+      slidingWindowSize: 100
```
Comment (Romeh) [inline resilience4j-spring-boot2/src/test/resources/application.yaml]: can we do the same for spring boo1 test resources as well

Comment (RobWin) [reply resilience4j-spring-boot2/src/test/resources/application.yaml]: Yes

Comment (RobWin) [reply resilience4j-spring-boot2/src/test/resources/application.yaml]: done

**Comment by RobWin** [comment]
Could you please review the changes again?

**Pull request merged**